projects/1.16/assets/betterend-re-forked/betterendforge/lang/zh_cn.json

```
@@ -0,0 +1,765 @@
+  "message.betterendforge.anvil_damage": "§c损坏",
+  "block.betterendforge.lumecorn": "发光玉米",
+  "block.betterendforge.lumecorn_seed": "发光玉米种子",
+  "item.betterendforge.lumecorn_rod": "发光玉米棒",
+  "block.betterendforge.blooming_cooksonia": "开花顶囊蕨",
+  "block.betterendforge.salteago": "林隙草",
+  "block.betterendforge.vaiolush_fern": "曲腿蕨",
+  "block.betterendforge.fracturn": "分裂草",
+  "biome.betterendforge.dry_shrubland": "干燥灌丛",
+  "biome.betterendforge.neon_oasis": "霓虹绿洲",
+  "block.betterendforge.neon_cactus": "霓虹仙人掌",
+  "block.betterendforge.neon_cactus_block": "霓虹仙人掌块",
+  "block.betterendforge.neon_cactus_slab": "霓虹仙人掌台阶",
+  "block.betterendforge.neon_cactus_stairs": "霓虹仙人掌楼梯",
+  "item.betterendforge.music_disc_strange_and_alien": "§b音乐唱片§r",
+  "item.betterendforge.music_disc_strange_and_alien.desc": "§5Firel§r - §fStrange And Alien§r",
+  "item.betterendforge.music_disc_grasping_at_stars": "§b音乐唱片§r",
+  "item.betterendforge.music_disc_grasping_at_stars.desc": "§5Firel§r - §fGrasping At Stars§r",
+  "item.betterendforge.music_disc_endseeker": "§b音乐唱片§r",
+  "item.betterendforge.music_disc_endseeker.desc": "§5Firel§r - §fEndseeker§r",
+  "item.betterendforge.music_disc_eo_dracona": "§b音乐唱片§r",
+  "item.betterendforge.music_disc_eo_dracona.desc": "§5Firel§r - §fEo Dracona§r",
+  "block.betterendforge.rutiscus": "芸香草方块",
+  "block.betterendforge.rutiscus_path": "芸香草径",
+  "biome.betterendforge.lantern_woods": "灯笼树林",
+  "block.betterendforge.filalux": "灯芯藤",
+  "block.betterendforge.filalux_lantern": "灯芯藤灯笼",
+  "block.betterendforge.filalux_wings": "灯芯藤翅",
+  "block.betterendforge.lucernia_bark": "灯笼木",
+  "block.betterendforge.lucernia_barrel": "灯笼木桶",
+  "block.betterendforge.lucernia_bookshelf": "灯笼木书架",
+  "block.betterendforge.lucernia_button": "灯笼木按钮",
+  "block.betterendforge.lucernia_chest": "灯笼木箱子",
+  "block.betterendforge.lucernia_composter": "灯笼木堆肥桶",
+  "block.betterendforge.lucernia_crafting_table": "灯笼木工作台",
+  "block.betterendforge.lucernia_door": "灯笼木门",
+  "block.betterendforge.lucernia_fence": "灯笼木栅栏",
+  "block.betterendforge.lucernia_gate": "灯笼木栅栏门",
+  "block.betterendforge.lucernia_ladder": "灯笼木梯子",
+  "block.betterendforge.lucernia_leaves": "灯笼树叶",
+  "block.betterendforge.lucernia_log": "灯笼原木",
+  "block.betterendforge.lucernia_outer_leaves": "灯笼外层树叶",
+  "block.betterendforge.lucernia_planks": "灯笼木板",
+  "block.betterendforge.lucernia_pressure_plate": "灯笼木压力板",
+  "block.betterendforge.lucernia_sapling": "灯笼树苗",
+  "block.betterendforge.lucernia_sign": "灯笼木告示牌",
+  "block.betterendforge.lucernia_slab": "灯笼木台阶",
+  "block.betterendforge.lucernia_stairs": "灯笼木楼梯",
+  "block.betterendforge.lucernia_stripped_bark": "去皮灯笼木",
+  "block.betterendforge.lucernia_stripped_log": "去皮灯笼原木",
+  "block.betterendforge.lucernia_trapdoor": "灯笼木活板门",
+  "block.betterendforge.flamaea": "火焰莲",
+  "block.betterendforge.aeridium": "指甲兰",
+  "block.betterendforge.lamellarium": "片状草",
```

```suggestion
  "block.betterendforge.lamellarium": "片状草方块",
```
如果是如同草方块的地质方块的话
上面的`芸香草`可能没表达清楚，抱歉

---

这个是植物，芸香草那个是方块，然后还有暗影草方块

---

只要看有没有对应的草径就好了...应该没了